File: HMP.js
const map = new Map();
map.set("", "");
map.set("", "");
map.set("", "");
console.log(map);

// if the map contains the key, get it and print it
if (map.has("Key1")) {
    const a = map.get("Key1");
    console.log(`value for key "Key1" is: ${a}`);
}

File: HMP.java
import java.util.HashMap;
public class HMP {
    public static void main(String[] args)
    {
        // making a hash map with 2 strings here 
        HashMap<String, String> map = new HashMap<>();
        map.put("", "");
        map.put("", "");
        map.put("", "");
        System.out.println(map);
        // if the map contains the key, get it and print it
        if (map.containsKey("Key1")) {
            String a = map.get("Key1");
            System.out.println("value for key"
                               + " \"Key1\" is: " + a);
        }
    }
}
